pkg/validator/sarif.go

```diff
@@ -0,0 +1,17 @@
+package validator
+
+import (
+	"github.com/owenrumney/go-sarif/v2/sarif"
+)
+
+type SarifValidator struct{}
+
+// Validate implements the Validator interface by attempting to
+// parse .sarif file using UnMarshal.json
+func (SarifValidator) Validate(b []byte) (bool, error) {
+	_, err := sarif.FromBytes(b)
+	if err != nil {
+		return false, err
```

Comment:
Since `sarif.FromBytes` does `json.Unmarshal`, we can return a custom error from JSON validator, which will explicitly display the precise location in the file where error occurred.

Reply:
```golang
func (SARIFValidator) Validate(b []byte) (bool, error) {
	_, err := sarif.FromBytes(b)
	if err != nil {
		customError := getCustomErr(b, err)
		return false, customError
	}
```

Yeah, if return value from `sarif.FromBytes` (except for the error) isn't needed, it's possible just to use JSON validator instead.
```golang
func (SARIFValidator) Validate(b []byte) (bool, error) {
	var output any
	err := json.Unmarshal(b, &output)
	if err != nil {
		customError := getCustomErr(b, err)
		return false, customError
	}
	return true, nil
}

```